docs/content/guides/dagster/run-attribution.mdx

```diff
@@ -21,7 +21,7 @@ In this use case, we'd like to add a hook to customize submitted runs while stil
 
 ```python file=/guides/dagster/run_attribution/custom_run_coordinator_skeleton.py startafter=start_custom_run_coordinator_marker endbefore=end_custom_run_coordinator_marker
 from dagster._core.run_coordinator import QueuedRunCoordinator, SubmitRunContext
-from dagster._core.storage.pipeline_run import DagsterRun
```

Comment:
Theres a top level export for this right? should probably switch to that. 

Are we preserving something at `pipeline_run` to prevent breaking users who referenced our documentation directly?

Reply:
Good point. I just pushed an entry that aliases `pipeline_run` in the top-level `dagster/__init__.py` module alias table that will prevent this import from breaking (I tested it manually).

I will put up another PR that removes all non-top-level imports in `docs_snippets`, we shouldn't have those.